### PR TITLE
Temperature Unit Conversion [FEATURE]

### PR DESCRIPTION
**Pull request by b1nary-c0de**
## Description

Converts Temperature from Celsius to Fahrenheit and also from Fahrenheit to Celsius.

## What does your PR belong to?
- [ ] Website
- [X] Snippets
- [ ] General / Things regarding the repository (like CI Integration)

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (non-breaking improvement of a snippet)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly. 
- [X] I have checked that the changes are working properly
- [X] I have checked that there isn't any PR doing the same
- [X] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: snippets/celsius_to_fahrenheit.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: celsius_to_fahrenheit
+tags: conversion, beginner
+---
+
+Converts Celsius to Fahrenheit.
+
+fahrenheit = (celsius * 1.8) + 32
```
Comment (Chalarangelo) [inline snippets/celsius_to_fahrenheit.md:8]: Please make your description more like a sentence (e.g. `Use the formula ... to convert the temperature from Celsius to Fahrenheit.`)

File: snippets/fahrenheit_to_celsius.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+title: fahrenheit_to_celsius
+tags: conversion, beginner
+---
+
+Converts Fahrenheit to Celsius.
+
+celsius = (fahrenheit - 32) / 1.8
```
Comment (Chalarangelo) [inline snippets/fahrenheit_to_celsius.md:8]: Same as the other snippet, make this a sentence, please.

File: snippets/celsius_to_fahrenheit.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+---
+title: celsius_to_fahrenheit
+tags: conversion, beginner
```
Comment (Chalarangelo) [inline snippets/celsius_to_fahrenheit.md:3]: Tags should be `math,beginner` as we do not have a conversion category and `math` is where we usually put these kinds of snippets.

File: snippets/fahrenheit_to_celsius.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+---
+title: fahrenheit_to_celsius
+tags: conversion, beginner
```
Comment (Chalarangelo) [inline snippets/fahrenheit_to_celsius.md:3]: Tags should be `math,beginner` as we do not have a conversion category and `math` is where we usually put these kinds of snippets.

File: snippets/celsius_to_fahrenheit.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+fahrenheit = (celsius * 1.8) + 32
+
+```py
+def celsius_to_fahrenheit(celsius):
+  return ((celsius * 1.8) + 32)
+```
+
+```py
+celsius_to_fahrenheit(25) # 77.0
```
Comment (Chalarangelo) [inline snippets/celsius_to_fahrenheit.md:16]: No need for that many examples, one will suffice as it's a straightforward use-case.

File: snippets/fahrenheit_to_celsius.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+
+celsius = (fahrenheit - 32) / 1.8
+
+```py
+def fahrenheit_to_celsius(fahrenheit):
+  return ((fahrenheit - 32) / 1.8)
+```
+
+```py
+fahrenheit_to_celsius(77) # 25.0
```
Comment (Chalarangelo) [inline snippets/fahrenheit_to_celsius.md:16]: No need for that many examples, one will suffice as it's a straightforward use-case.

**Comment by b1nary-c0de** [comment]
Thanks for the feedback. I have corrected the snippets accordingly.

**Pull request merged**